include/settings/settings.h

```diff
@@ -93,6 +149,28 @@ struct settings_handler {
 	 */
 };
 
+/**
+ * Define a static handler for settings items
+ *
```

Comment:
Just a minor addition here, you might want to add that us the variable name the macros expands to just in case the code wants to make use of it for some reason/

Reply:
@Vudentz, thanks. I don't quite understand your last addition. Can I return the created variable name ? And how would it be used (I am not asking about the why it would be used) ? My knowledge of c (and macro's) is still limited but I'm always open to learn.